lib/keen/UiTextbox.vue

```diff
@@ -90,7 +90,7 @@
         </div>
       </label>
 
-      <div v-if="showHelp || hasErrorFeedback || maxlength" class="ui-textbox-feedback">
+      <div v-show="showHelp || hasErrorFeedback || maxlength" class="ui-textbox-feedback">
```

Comment:
Was curious if we could simply remove the conditional on this altogether?

```suggestion
      <div class="ui-textbox-feedback">
```

It's not entirely clear to me why it exists, and removing it would consistently preserve space for validation messages.